button/stories/button.stories.tsx

```diff
@@ -1,13 +1,50 @@
-import type { Meta } from '@storybook/react';
-import type { ButtonProps } from '@mantine/core';
-import { Button } from '@mantine/core';
+import type { Meta, StoryObj } from '@storybook/react';
+import { iconNames } from '../../icon/src/icon';
+import { Button, ButtonProps } from '../src/button';
 
-export default {
+const meta: Meta = {
   title: 'Buttons/Button',
   component: Button,
+};
+
+export default meta;
+
+export const Playground: StoryObj<ButtonProps> = {
+  argTypes: {
+    children: { type: { name: 'string', required: true }, control: 'text' },
+    variant: {
+      type: { name: 'string', required: true },
+      control: 'select',
+      options: ['primary', 'secondary'],
+    },
+    compact: { type: { name: 'boolean', required: true }, control: 'boolean' },
+    disabled: { type: { name: 'boolean', required: true }, control: 'boolean' },
+    iconPosition: {
+      type: { name: 'string', required: true },
+      options: ['none', 'left', 'right'],
+      control: { type: 'select' },
+    },
+    icon: {
+      type: { name: 'string', required: false },
+      options: [iconNames[0]],
```

Comment:
this is IMO better than a `containsIcon` boolean I saw in other storybooks, because
- boolean means 2 variants multiplied by 2 icon positions `left` and `right` which gives a multiplier of `2x2=4`
- just an icon string (first icon name is hardcoded) gives 1 variant multiplied by 3 icon positions `none` (no icon), `left` and `right` which gives a multiplier `1x3=3`

cc @gluck @thais-divriots